posts/2022-03-24 adding a file via SFTP to Vendia Share

```diff
@@ -0,0 +1,20 @@
+title: 'Add a file via SFTP to Vendia Share'
+description: 'Send a CSV over a SFTP to add data to your Uni'
+date: '2022-03-23'
+thumbnail: 'http://url-to-thumbnail.jpg'
+authors:
+  - Caroll Casbeer
+
+In Part 1 of our series about Integrating Data Stores with Vendia Share, we dove into adding a [CSV via Email](https://www.vendia.net/blog/add-csv-to-vendia-share), but what about adding data via STFP? We can help with that!
+
+## What’s an SFTP?
+
+A Secure File Transfer Protocol (SFTP) provides reliable data and large file transfers over the web. SFTP is regularly used when transferring sensitive information like PCI, PHI, financial records, or research. Thus it is no surprise that STFP is the standard for financial systems, and health services that have to comply with HIPAA.   
+## SFTP in Health and Financial Services
+For example, let’s say a hospital needs to relay results from a sponsored study to their partnered pharmaceutical company. They can’t just email it - it could be hacked. Instead, they use a STFP to transfer the data and relevant files. The STFP ensures a secure and accurate transference of sensitive information.
```

Comment:
```suggestion

For example, let’s say a hospital needs to relay results from a sponsored study to their partnered pharmaceutical company. They can’t just email it - it could be hacked. Instead, they use a STFP to transfer the data and relevant files. The STFP ensures a secure and accurate transference of sensitive information.
```